File: king-admin-java/src/main/java/com/oukingtim/domain/SysUser.java
package com.oukingtim.domain;

import com.baomidou.mybatisplus.annotations.TableField;
import com.baomidou.mybatisplus.annotations.TableName;
import lombok.Data;
import lombok.EqualsAndHashCode;

import java.util.List;

/**
 * Created by oukingtim
 */
@TableName("sys_user")
@Data
@EqualsAndHashCode(callSuper = false)
public class SysUser extends BaseModel<SysUser> {

    private String username;
    private String password;
    private String email;
    private String mobile;
    private String status;

    @TableField(exist=false)
    private List<SysRole> rolelist;


    public SysUser(String username) {
        this.username = username;
    }

    public SysUser() {
    }
}

File: king-admin-kotlin/src/main/kotlin/com/oukingtim/domain/SysUser.kt
package com.oukingtim.domain

import com.baomidou.mybatisplus.annotations.TableField
import com.baomidou.mybatisplus.annotations.TableName

/**
 * Created by oukingtim
 */
@TableName("sys_user")
class SysUser() : BaseModel<SysUser>() {

    var username: String? = null
    var password: String? = null
    var email: String? = null
    val mobile: String? = null
    val status: String? = null

    @TableField(exist = false)
    var rolelist: List<SysRole>? = null

    constructor(username: String) :this(){
        this.username = username
    }

}